example/proguard-rules.pro

```diff
@@ -15,3 +15,6 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
 #   public *;
 #}
+
+-keep class com.zomato.photofilters.** {*;}
+-keepclassmembers  class com.zomato.photofilters.** {*;}
```

Comment:
This is missing a new line at the EOF. Can you please add that?

Reply:
Sure. This has been solved in the latest commit.